app/templates/search/_filters.html

```diff
@@ -1,13 +1,18 @@
 <div class="dm-filters">
-  <h2 class="apply-filters-title">Apply filters</h2>
-  {% for filter in filters %}
-    {%
-      with
-      label = filter.label,
-      options = filter.filters
-    %}
-      {% include "toolkit/forms/option-select.html" %}
-    {% endwith %}
-  {% endfor %}
+  <div class="dm-filter-title">
+    <h2 class="apply-filters-title">Apply filters</h2>
+    <a id="clearAllFilters" class="clear-filters-link" href="{{ clear_filters_url }}">Clear filters</a>
```

Comment:
This ID still needs updating to `clear-all-filters` or `dm-clear-all-filters` (and respectively in the frontend-toolkit PR).